bigtable/client/filters.h

```diff
@@ -215,17 +218,17 @@ class Filter {
   /**
    * Return a filter that skips the first @p n cells in a row.
    *
-   * Notice that cells might be repeated, such as when interleaving the results
-   * of multiple filters via the Union() function (aka Interleaved in the
-   * proto).  Furthermore, notice that this is the cells within a row, if there
-   * are multiple column families and columns, the cells are returned ordered
-   * by:
-   * - The column family internal ID, which is not necessarily the
-   *   lexicographical order of the column family names.
-   * - The column names, lexicographically.
-   * - Timestamp in reverse order.
-   *
-   * TODO(#82) - check the documentation around ordering of columns.
+   * Note that cells might be repeated, such as when interleaving the results
+   * of multiple filters via the `Interleave()` function.  Furthermore, this
+   * filter apples to the cells within a row, if there are multiple column
+   * families and/or columns in a row the order is:
+   * - All the cells for a column family appear together, but there is no
```

Comment:
Same comment about rendering as above.

Reply:
I did, it works.  Look at the attached file:

[filter-doxygen-render.pdf](https://github.com/GoogleCloudPlatform/google-cloud-cpp/files/1591934/filter-doxygen-render.pdf)

Reply:
Perfect, thanks!